### PR TITLE
Add test for worker thread support

### PR DESCRIPTION
**Pull request by Widdershin**
Just a simple test, loading the module, capturing a small profile and
sending it back over the wire.

The test will be skipped if the worker_threads module can't be required.

### CODE REVIEW

**Review by hyj1991 — APPROVED** [review]
+1

File: test/v8-profiler.js [hunk 1]
```diff
@@ -1,7 +1,18 @@
-const expect  = require('chai').expect,
+"use strict";
+const expect = require('chai').expect,
+      path = require('path'),
       profiler = require('../');
 
 const NODE_V_010 = /^v0\.10\.\d+$/.test(process.version);
+const SOURCE_PATH = path.join(__dirname, '..');
+
+var workerThreads;
```
Comment (hyj1991) [inline test/v8-profiler.js]: Here could be `let`, but I think we could open another pr to fix it because of many other `var` in the raw file.

Comment (Widdershin) [reply test/v8-profiler.js]: Breaks Node 4 and 5 on Travis

Comment (Widdershin) [reply test/v8-profiler.js]: Actually might just need a "use strict" in this file.

Comment (hyj1991) [reply test/v8-profiler.js]: > Actually might just need a "use strict" in this file.

enen

**Pull request merged**